### PR TITLE
Feat/env variable login

### PR DESCRIPTION
**Pull request by sp-daniel-sanchez**
Now you can load the Service using the config json file or env variables adding `from_env=True` when creating the Sheets or Drive service.

### CODE REVIEW

**Comment by SP-alex-muelas** [comment]
The code is failing for Python 2.7, which is a supported version for sheetfu:

https://travis-ci.org/socialpoint-labs/sheetfu/jobs/532183648

**Comment by sp-daniel-sanchez** [comment]
> The code is failing for Python 2.7, which is a supported version for sheetfu:
> 
> https://travis-ci.org/socialpoint-labs/sheetfu/jobs/532183648

Yes, i'm checking it right now, thanks!

File: sheetfu/service.py [hunk 1]
```diff
@@ -41,35 +44,54 @@ def build(self, http=None):
...
 
-    """Class to create ready to use drive service with service to service credentials"""
+    """Class to create ready to use sheets service with service to service credentials"""
+    SERVICE = "sheets"
+    VERSION = "v4"
 
-    def __init__(self, path_to_secret=None):
+    def __init__(self, path_to_secret=None, from_env=False):
         """
         :param path_to_secret: path to service to service json credentials file.
```
Comment (SP-alex-muelas) [inline sheetfu/service.py]: Add from_env parameter to docstring

File: sheetfu/service.py [hunk 2]
```diff
@@ -41,35 +44,54 @@ def build(self, http=None):
...
+    """Class to create ready to use drive service with service to service credentials"""
+    SERVICE = "drive"
+    VERSION = "v3"
+
+    def __init__(self, path_to_secret=None, from_env=False):
         """
-        Builds a sheet service, ready to query sheets.
-        :param http: Only needed for unit testing. Must be an object of HttpMockSequence. DO NOT USE IN PROD.
-        :return: An authorized sheets service.
+        :param path_to_secret: path to service to service json credentials file.
```
Comment (SP-alex-muelas) [inline sheetfu/service.py]: Same here, add to docstring

File: sheetfu/service.py [hunk 3]
```diff
@@ -8,30 +8,33 @@
...
-            'https://www.googleapis.com/auth/drive',
-            'https://www.googleapis.com/auth/spreadsheets'
-        ]
-        self.path_to_secret = path_to_secret
-        if self.path_to_secret is None:
-            self.credentials = None
+    def __init__(self, path_to_secret=None, from_env=False):
+        if from_env:
+            config_dict = self._build_keyfile_dict()
+            print(config_dict)
```
Comment (SP-alex-muelas) [inline sheetfu/service.py]: Remove print

File: sheetfu/client.py [hunk 4]
```diff
@@ -17,7 +17,7 @@
 
 class SpreadsheetApp:
 
-    def __init__(self, path_to_secret=None, http=None):
+    def __init__(self, path_to_secret=None, http=None, from_env=False):
```
Comment (SP-alex-muelas) [inline sheetfu/client.py]: Same here, add to docstring

**Pull request merged**